src/components/editor-page/document-bar/document-info/document-info-line-word-count.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useEffect, useState } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { ShowIf } from '../../../common/show-if/show-if'
+import { DocumentInfoLine } from './document-info-line'
+import { UnitalicBoldText } from './unitalic-bold-text'
+import { useIFrameEditorToRendererCommunicator } from '../../render-context/iframe-editor-to-renderer-communicator-context-provider'
+
+export const DocumentInfoLineWordCount: React.FC = () => {
+  useTranslation()
+  const iframeEditorToRendererCommunicator = useIFrameEditorToRendererCommunicator()
+
+  // -1 means the word count isn't determined yet, any non-negative number is the actual counting result
+  const [wordCount, setWordCount] = useState(-1)
```

Comment:
1. Magic number. Use a const for that.
2. Why don't you use undefined or null for that?

Reply:
changed